docs/app/styles/styles-routes.ts

```diff
@@ -36,6 +37,11 @@ export const routes: Routes = [
                 component: CodeDemoComponent,
                 data: { title: 'Code' }
             },
+            {
```

Comment:
The components are alphabetized. Should the styles be too?

Reply:
At first I was thinking we'd manually order them, but now that you mention it, we have enough in there that we probably should alphabetize them.  I'll make that change.